.claude/scripts/latency-discipline-contract.test.sh

```
@@ -0,0 +1,98 @@
+assert_bullet 'local timer for a process whose completion nothing will report' \
+  "the sleep carve-out is not scoped by whether completion is reported"
+
+# 3. BOTH-HALVES. The superseded absolute must be GONE, not merely supplemented. This is the
+#    assertion that actually fixes the measured defect: while this phrase survives, an agent reading
+#    the paragraph still finds a sufficient licence for polling a backgrounded task.
+refute_bullet 'a process you yourself started' \
+  "the superseded carve-out ('a process you yourself started') is still present — it licenses polling a backgrounded task whose completion the runtime reports, which is the shape the guard blocks"
+
+# 4. The correct primitive is named. A prohibition that does not say what to do instead is the
+#    DevEx tax this repo's own hardening rule forbids, and the guard's refusal already names it.
+assert_bullet '`Monitor` with an until-loop' \
```

Shellcheck (0.11.0)

[info] Expressions don't expand in single quotes, use double quotes for that.

(SC2016)